workflows/iuf/operations/ims-upload.yaml

```diff
@@ -0,0 +1,224 @@
+          - name: product-version
+            value: "{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.manifest.version')}}"
+          - name: yaml-content
+            value: "{{steps.ims-upload-content.outputs.parameters.ims_records}}"
+### Templates ###
+## get-s3-secrets-template ##
+  - name: get-s3-secrets-template
+    nodeSelector:
+      kubernetes.io/hostname: ncn-m001
+    tolerations:
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    outputs:
+      parameters:
+        - name: secret_name
+          valueFrom:
+            path: /tmp/secret_name
+    retryStrategy:
+        limit: "2"
+        retryPolicy: "Always"
+        backoff:
+          duration: "10s" # Must be a string. Default unit is seconds. Could also be a Duration, e.g.: "2m", "6h", "1d"
+          factor: "2"
+          maxDuration: "1m"
+    script:
+      image: artifactory.algol60.net/csm-docker/stable/docker.io/portainer/kubectl-shell:latest-v1.21.1-amd64
+      command: [bash]
+      source: |
+        function sync_item() {
+          item_name="$1"
+          source_ns="$2"
+          destination_name="$3-$RANDOM"
+          destination_ns="$4"
+          if kubectl get $item_name -n $source_ns &> /dev/null; then
+            echo "Syncing $item_name from $source_ns to $destination_ns as $destination_name"
+            kubectl get $item_name -n $source_ns -o json | \
+              jq 'del(.metadata.namespace)' | \
+              jq 'del(.metadata.creationTimestamp)' | \
+              jq 'del(.metadata.resourceVersion)' | \
+              jq 'del(.metadata.selfLink)' | \
+              jq 'del(.metadata.uid)' | \
+              jq 'del(.metadata.ownerReferences)' | \
+              jq 'del(.metadata.name)' | \
+              jq '.metadata |= . + {"name":"'$destination_name'"}' | \
+              kubectl apply -n $destination_ns -f -
+              return $?
+          else
+            echo "Didn't find $item_name in the $source_ns namespace"
+            return 1
+          fi
+        }
+        sync_item secret/ims-s3-credentials services ims-s3-credentials-argo argo
+        rc=$?
+        echo "$destination_name" >> /tmp/secret_name
+        exit $rc
+## ims-upload-content ##
+  - name: ims-upload-content
+    nodeSelector:
+      kubernetes.io/hostname: ncn-m001
+    tolerations:
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    inputs:
+      parameters:
+      - name: s3_credentials_secret_name
+      - name: ims_upload_ims_url
+        value: "https://api-gw-service-nmn.local/apis/ims/v2"
+      - name: ca_certs_bundle
+        value: /etc/ssl/certs/ca-certificates.crt
+      - name: auth_token
+      - name: iuf_release_mount_path
+      - name: media_dir
```

Comment:
I think this is no longer needed, along with ll. 51-52.

But then we do want to add `iuf_release_mount_path` as a volumeMount as we discussed in the meeting today.

Reply:
Looks like `iuf_release_mount_path` is addressed but I still think you could remove `media_dir` as a parameter now.